Sources/Plot/API/HTMLAttributes.swift

```diff
@@ -1,3 +1,4 @@
+import Foundation
```

Comment:
Remove this import please, as I'd prefer Plot to remain 100% dependency-free for maximum portability (this comment also applies to all other `import Foundation` statements within this PR).